### PR TITLE
Crash recovery docs

### PR DESCRIPTION
**Pull request by kacperzuk-neti**
In addition to crash recovery stuff, it also changes all docs to stop using JSON specs (as we're dropping them to make crash recovery easier).

### CODE REVIEW

**Comment by kacperzuk-neti** [comment]
PR for install.sh changes: https://github.com/liberland/liberland_substrate/pull/375

File: blockchain/for-validators-nominators-and-stakers/resync.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# Resyncing node
+
+To resync your Liberland Node, you need to:
+1. stop the node
```
Comment (DorianSternVukotic) [inline blockchain/for-validators-nominators-and-stakers/resync.md:4]: Missing step. Include stopping validating in blockchain.liberland.org or polkadotjs and waiting until you are not in active validator set

Comment (DorianSternVukotic) [reply blockchain/for-validators-nominators-and-stakers/resync.md:4]: add this step to other places where applicable

**Review by DorianSternVukotic — CHANGES_REQUESTED** [review]

File: blockchain/for-validators-nominators-and-stakers/resync.md (hunk 1 above)
Comment (kacperzuk-neti) [reply blockchain/for-validators-nominators-and-stakers/resync.md]: Done. Other places don't need it (this is the only instruction that causes prolonged downtime of the node).

**Comment by DorianSternVukotic** [comment]
conflicts

**Comment by kacperzuk-neti** [comment]
> conflicts

Fixed

**Pull request merged**